src/Exceptions/HttpUnsupportedMediaType.php

```diff
@@ -8,4 +8,3 @@ class HttpUnsupportedMediaType extends HttpException {
     public const DESCRIPTION = "Indicates that the origin server is refusing to service the request because the payload is in a format not supported by the target resource for this method.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/415";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces